apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,331 @@
+| **messageBatchDepth**   | Defines how many messages in a batch can the circuit process.                  |
+| **zkeys**               | Defines the path to the zkey files for QV and Non QV keys.                     |
+
+:::important
+The recommended values for test keys are: **10-1-2-2-1**. For ceremony keys: **14-5-9-3-2**.
+:::
+
+#### Poll
+
+| Property               | Description                                          |
+| ---------------------- | ---------------------------------------------------- |
+| **pollDuration**       | Defines how long is going to be the poll in seconds. |
+| **coordinatorPubkey**  | Defines the coordinator public MACI key.             |
+| **useQuadraticVoting** | Defines if the poll uses quadratic voting or not.    |
+
+### Deploy MACI Contracts
+
+To deploy the MACI contracts to a specific network you can append `:network` to the deployment commands, e.g. `pnpm deploy:sepolia` - please refer to the available networks on the package.json scripts section.
+
+```bash
+pnpm deploy:NETWORK
+```
+
+There are already some deployed contracts that could be reused, copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save your gas fee.
+
+```bash
+cp default-deployed-contracts.json deployed-contracts.json
+```
+
+Delete any contract you want to redeploy and then run the following command to save gas:
+
+```bash
+pnpm deploy:NETWORK --incremental
+```
+
+## Deploy Poll
+
+To deploy your first Poll you can run the following command:
+
+```sh
+pnpm deploy-poll:NETWORK
+```
+
+:::important
+Making another poll doesn't require deploying MACI contracts again, you can deploy another poll running the `pnpm deploy-poll:NETWORK` command and then use the updated poll-id.
+:::
+
+## Poll Finalization
+
+As a coordinator, first you need to merge signups and messages (votes). Messages are stored in a queue so when the poll is over, the coordinator needs to create the tree from the queue ([AccQueue](/docs/core-concepts/merkle-trees#accumulator-queue)). This optimization is needed to reduce gas cost for voters. Then coordinator generates proofs for the message processing, and tally calculations. This allows to publish the poll results on-chain and then everyone can verify the results when the poll is over. You run a merge with:
+
+```bash
+pnpm merge:[network] --poll [poll-id]
+```
+
+Then you need to generate the proofs for the message processing, and tally calculations, we do it with the following command:
+
+```bash
+pnpm run prove:[network] --poll [poll-id] \
+    --coordinator-private-key [coordinator-maci-private-key] \
+    --tally-file ../proofs/tally.json \
+    --output-dir ../proofs/proofs/ \
+    --start-block [block-number]  \
+    --blocks-per-batch [number-of-blocks]
+```
+
+:::important
+You can improve the time of the proving by including more blocks per batch, by default you can start with 500.
```

Comment:
"improve" -> "reduce"

"You can reduce the proving time by increasing the blocks per batch parameter"

Reply:
is it 500 by default?

Reply:
Not by default; It's just a recommendation, will make it clear